Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -1196,14 +1196,14 @@ public function isBinaryOperator($index)
                 T_XOR_EQUAL => true,            // ^=
             );
             if (defined('T_POW')) {
-                $arrayOperators[T_POW] = true;          // **
-                $arrayOperators[T_POW_EQUAL] = true;    // **=
+                $arrayOperators[T_POW] = true; // **
+                $arrayOperators[T_POW_EQUAL] = true; // **=
```

Comment:
I'll update to check for a comment after the whitespace, otherwise this cannot be formatted to PSR2

```php
        for (;   $u3 < 3;   ++$u3) {
        }
```
to
```php
        for (; $u3 < 3; ++$u3) {
        }
```